conf_files/pocs.yaml

```diff
@@ -115,3 +115,24 @@ panoptes_network:
 #         output_timestamp: False
 
 state_machine: simple_state_table
+
+########################### Flat Fields ########################################
+# Controls various aspects of flat fields, both for evening and morning. 
+# 
+# Note: The times at which flat-fielding will occur are controlled by the
+# `location.flat_horizon` and `location.observe_horizon` config entries.
+#
+# take_evening_flats: If we should try to take flats during evening twilight
+# take_morning_flats: If we should try to take flats during morning twilight
+# evening:            The coordinates should be roughly opposite the sun.
+# morning:            The coordinates should be roughly opposite the sun.
+################################################################################
+flat_field:
+    take_evening_flats: False
+    take_morning_flats: False
+    evening:
+        alt: 70  # degrees
```

Comment:
There are actually straight from Huntsman. The idea is it should be roughly opposite the setting sun, so we can (and probably should) compute these for each unit.

I changed the `alt` to 50° for PAN001 to test it out but no good weather yet.

Reply:
The idea here is to chose flat fielding coordinates so that the gradient in the twilight sky brightness across the field of view is both minimised and kept as consistent as possible. That way it is possible to take advantage of the meridian flip of a German equatorial mount by combining twilight flats from evening and morning twilight, thereby cancelling out the sky gradients.

To do this the azimuth should be computed so that it is opposite the Sun at the unit's location at the time the flats are taken. The altitude can be set in the config file, and the best values are probably fairly close to zenith.

Reply:
I've added an issue in #732 to adjust changing the azimuth.